modules/era/sql/synth_recipes_era.sql

```diff
@@ -18,453 +18,3476 @@ ALTER TABLE `synth_recipes`
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bronze Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Brass Spear';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bouncer Club (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Boomerang';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Book Holder';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bone Arrow';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bolt Belt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bloody Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bloody Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Blind Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Blind Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bay Aquarium';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bastokan Crossbow';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bamboo Fishing Rod';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Bamboo Fishing Rod';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Staff';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Staff (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Pole';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Pole (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Club';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Club (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ash Clogs';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Arrowwood Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Arrowwood Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Angel\'s Flute';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Angel\'s Flute (desynth)';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ancient Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Ancient Lumber';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt';
-UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt';
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acheron Shield' AND ID = 54538;
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt' AND ID = 529;
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Acid Bolt' AND ID = 531;
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Adaman Cuirass' AND ID = 15031;
+UPDATE `synth_recipes` SET ContentTag='COP' WHERE ResultName='Adaman Cuisses' AND ID = 14574;
```

Comment:
I don't think this recipe exists, it's the same as the 15031 recipe just with less of each ingredient